js/ebanx/document/document.js

```diff
@@ -25,11 +26,15 @@ var changeTaxVatLabel = function () {
   var newLabel = getLabelByCountry(country, defaultLabel);
 
   taxVatLabel.innerHTML = newLabel;
+    if(taxVatInput) {
+      setTimeout(function(){taxVatInput.placeholder = newLabel;}, 10)
```

Comment:
Magento has some code that generates the checkout fields again, so with setTimeout we solve this.